RecoBTag/DeepFlavour/python/DeepFlavour_cff.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
pfDeepFlavour_cff ?
unless there is a plan to have more stuff here

Reply:
will change

Reply:
changed with https://github.com/cms-sw/cmssw/pull/19893/commits/acdfb394d2633ca089787cf8969a18d536eea3a8